src/router.js

```diff
@@ -0,0 +1,12 @@
+import VueRouter from 'vue-router';
+import Albums from './components/Albums';
+import AlbumDetail from './components/AlbumDetail';
+import Home from './Home/Home';
```

Comment:
I don't recommend changing it this time, but usually folders are lowercased. If you do want to rename the folder, I can help you with that. Just let me know!
```suggestion
import Home from './home/Home';
```